test/deprecated/atomic/return-by-ref-array.compopts

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Is this extra line intentional?  Does it run something?

Reply:
Ditto at least the single version as well, if something needs to change

Reply:
I don't think it runs anything, but I did remove it